TRAIN.md

```diff
@@ -223,3 +223,17 @@ EvalAI expects results to be structured in a specific format for a submission to
     ```
    The final submission file will be written to `is2rs_submission.npz` (rename accordingly).
 3. Upload `is2rs_submission.npz` to EvalAI.
+
+
+## Create EvalAI OC22 submission files
+
+EvalAI expects results to be structured in a specific format for a submission to be successful. A submission must contain results from the 2 different splits - in distribution (id) and out of distribution (ood). Construct submission files for the S2EF/IS2RE tasks as follows:
+
+### S2EF/IS2RE:
+1. Run predictions `--mode predict` on both the id and ood splits, generating `[s2ef/is2re]_predictions.npz` files for each split.
+2. Run the following command:
+    ```
+    python make_submission_file.py --oc22 --id path/to/id/file.npz --ood path/to/ood_ads/file.npz --out-path submission_file.npz
```

Comment:
Should we explicitly say here what the "oc22" flag does?

Reply:
I added a description of the --dataset flag